features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -232,6 +238,102 @@ internal open class RumViewScope(
         }
     }
 
+    private fun onStartFeatureOperation(event: RumRawEvent.StartFeatureOperation, writer: DataWriter<Any>) {
+        sdkCore.newRumEventWriteOperation(writer) { datadogContext ->
+            newVitalEvent(
+                datadogContext,
+                name = event.name,
+                operationKey = event.operationKey,
+                stepType = RumVitalEvent.StepType.START,
+                failureReason = null,
+                attributes = event.attributes
+            )
+        }.submit()
+        sendViewUpdate(event, writer)
+    }
+
+    private fun onStopFeatureOperation(event: RumRawEvent.StopFeatureOperation, writer: DataWriter<Any>) {
+        sdkCore.newRumEventWriteOperation(writer) { datadogContext ->
+            newVitalEvent(
+                datadogContext,
+                name = event.name,
+                operationKey = event.operationKey,
+                stepType = RumVitalEvent.StepType.END,
+                failureReason = event.failureReason?.toSchemaFailureReason(),
+                attributes = event.attributes
+            )
+        }.submit()
+        sendViewUpdate(event, writer)
```

Comment:
why do we need to send view update if creating/sending vital event doesn't change any state (at least of `RumViewScope`; it is clear that it _might_ change the state of upstream scopes)?

do we risk exploding the number of view updates?